src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractClassNameCheck.java

```diff
@@ -52,10 +52,23 @@
  * {@code false}.</li>
  * </ul>
  * <p>
- * The following example shows how to configure the {@code AbstractClassName} to
- * checks names, but ignore missing {@code abstract} modifiers:
+ * How to configure the {@code AbstractClassName} to use
```

Comment:
The common wording for default config is
` To configure the check: `
Please take a look over existent [docs](https://checkstyle.org/config_coding.html)

Reply:
done